code/game/objects/items/devices/radio/radio.dm

```diff
@@ -317,17 +317,20 @@
 	talk_into(speaker, raw_message, , spans, language=message_language)
 
 // Checks if this radio can receive on the given frequency.
-/obj/item/radio/proc/can_receive(freq, level)
+/obj/item/radio/proc/can_receive(freq, levels)
 	// deny checks
 	if (!on || !listening || wires.is_cut(WIRE_RX))
 		return FALSE
 	if (freq == FREQ_SYNDICATE && !syndie)
 		return FALSE
+	//Exploration signals can always be recieved
+	if (freq == FREQ_EXPLORATION)
+		return TRUE
```

Comment:
Not sure how I feel about this. I think they should be "stranded" if station tcomms are down.

Reply:
messages can only be received from anywhere, but cannot be sent. Like how when the station tcomms goes down your headset can still receive subspace messages.

Reply:
Ah that's fine then.